chainer/functions/array/split_axis.py

```diff
@@ -13,8 +15,14 @@
 _numpy_split_ok = numpy.lib.NumpyVersion(numpy.__version__) >= '1.11.0'
 
 
-def _fix_numpy_split(ys, x, indices_or_sections, axis):
+def _numpy_1_11_split(x, indices_or_sections, axis):
     """Make the output of np.split compatible with numpy >= 1.11"""
+    with warnings.catch_warnings():
```

Comment:
Use of `catch_warnings` should be avoided because it is thread unsafe.

As this affects users using NumPy 1.10.x or earlier, I feel it's ok to leave as it is.